bach/bach/series/series.py

```diff
@@ -838,13 +854,25 @@ def reset_index(
 
         return result
 
-    def sort_values(self, *, ascending=True):
+    def sort_values(self, *, ascending=True, keys: Optional[List['Series']] = None):
         """
         Sort this Series by its values.
         Returns a new instance and does not actually modify the instance it is called on.
 
         :param ascending: Whether to sort ascending (True) or descending (False)
+        :param keys: Apply sorting based on expressions from other series that share same
+        base node as caller
         """
+        if keys and any(key.base_node != self.base_node for key in keys):
+            raise ValueError('All series provided as keys must have the same base_node as caller.')
+
+        if keys:
```

Comment:
```suggestion
        if keys is not None:
```
This would allow one to set the sorting to 'no sorting' by passing `[]`. Otherwise there is no way to remove sorting

Reply:
Changed https://github.com/objectiv/objectiv-analytics/pull/995/commits/7b41884be66ed08c546c7cde09e2379398918f37